### PR TITLE
Added iteration number to stdout

### PR DESCRIPTION
**Pull request by farquet**
Added an `iteration` column to the slow cooker stdout in order to know how many iterations to easily see the current progress of the benchmark.
```
#                         iteration good/b/f t   goal%  min [p50 p95 p99  p999]  max bhash change
2018-10-08T08:28:24+02:00 0    382/0/166 12500   3% 5s   4 [  6  12 2305 2307 ] 2306      0 +
2018-10-08T08:28:29+02:00 1   6493/0/0 12500  51% 5s   2 [  3   5   7   18 ]   18      0 -
2018-10-08T08:28:34+02:00 2   7834/0/0 12500  62% 5s   1 [  3   3   4   20 ]   20      0 --
2018-10-08T08:28:39+02:00 3   1689/15/0 12500  13% 5s   1 [  2   3  15 1058 ] 1058      0 -
2018-10-08T08:28:44+02:00 4     33/10/0 12500   0% 5s   1 [  2 1007 2008 2008 ] 2008      0
2018-10-08T08:28:49+02:00 5     28/25/0 12500   0% 5s   1 [  3 1010 2007 4011 ] 4011      0
2018-10-08T08:28:54+02:00 6     27/2/0 12500   0% 5s   1 [  2 3007 3007 3007 ] 3007      0
2018-10-08T08:28:59+02:00 7   4705/10/0 12500  37% 5s   0 [  2   3   4 2007 ] 2007      0 --
2018-10-08T08:29:04+02:00 8   6753/0/0 12500  54% 5s   0 [  1   2   3  662 ]  662      0 --
2018-10-08T08:29:09+02:00 9   4771/0/0 12500  38% 5s   0 [  1   2   2 2008 ] 2008      0 --
...
```

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -403,7 +405,7 @@ func main() {
 		fmt.Printf("# sending %d %s req/s with concurrency=%d using url list %s ...\n", (*qps * *concurrency), *method, *concurrency, urldest[1:])
 	}
 
-	fmt.Printf("# %s good/b/f t   goal%% %s min [p50 p95 p99  p999]  max bhash change\n", timePadding, intPadding)
+	fmt.Printf("# %s iteration good/b/f t   goal%% %s min [p50 p95 p99  p999]  max bhash change\n", timePadding, intPadding)
```
Comment (siggy) [inline main.go]: In order to keep the alignment correct for the default case, would you mind adding a few more formatting changes...

```diff
diff --git a/main.go b/main.go
index 1091ad5..90b225f 100644
--- a/main.go
+++ b/main.go
@@ -395,7 +395,7 @@ func main() {
 	var sendTraffic sync.WaitGroup
 	// The time portion of the header can change due to timezone.
 	timeLen := len(time.Now().Format(time.RFC3339))
-	timePadding := strings.Repeat(" ", timeLen)
+	timePadding := strings.Repeat(" ", timeLen-len("# "))
 	intLen := len(fmt.Sprintf("%s", *interval))
 	intPadding := strings.Repeat(" ", intLen-2)
 
@@ -405,7 +405,7 @@ func main() {
 		fmt.Printf("# sending %d %s req/s with concurrency=%d using url list %s ...\n", (*qps * *concurrency), *method, *concurrency, urldest[1:])
 	}
 
-	fmt.Printf("# %s iteration good/b/f t   goal%% %s min [p50 p95 p99  p999]  max bhash change\n", timePadding, intPadding)
+	fmt.Printf("# %s iter   good/b/f t   goal%% %s min [p50 p95 p99  p999]  max bhash change\n", timePadding, intPadding)
 	stride := *concurrency
 	if stride > len(dstURLs) {
 		stride = 1
@@ -494,7 +494,7 @@ func main() {
 			changeIndicator := window.CalculateChangeIndicator(latencyHistory.Items, lastP99)
 			latencyHistory.Push(lastP99)
 
-			fmt.Printf("%s %d %6d/%1d/%1d %d %3d%% %s %3d [%3d %3d %3d %4d ] %4d %6d %s\n",
+			fmt.Printf("%s %4d %6d/%1d/%1d %d %3d%% %s %3d [%3d %3d %3d %4d ] %4d %6d %s\n",
 				t.Format(time.RFC3339),
 				iteration,
 				good,
```

For reference, here's the existing output on master:
```
# sending 1 GET req/s with concurrency=1 to http://localhost:4140/ ...
#                           good/b/f t   goal%   min [p50 p95 p99  p999]  max bhash change
2018-10-08T09:43:31-07:00      9/0/0 10  90% 10s  11 [ 13  70  70   70 ]   70      0 +
2018-10-08T09:43:41-07:00     10/0/0 10 100% 10s  11 [ 12  15  15   15 ]   15      0
2018-10-08T09:43:51-07:00     10/0/0 10 100% 10s  10 [ 13  15  15   15 ]   15      0
```
...and here's the output with the above diff applie:
```
# sending 1 GET req/s with concurrency=1 to http://localhost:4140/ ...
#                         iter   good/b/f t   goal%   min [p50 p95 p99  p999]  max bhash change
2018-10-08T10:23:14-07:00    0      9/0/0 10  90% 10s  10 [ 11  13  13   13 ]   13      0 +
2018-10-08T10:23:24-07:00    1     10/0/0 10 100% 10s  11 [ 11  14  14   14 ]   14      0
```

**Comment by farquet** [comment]
Formatting changes done

**Comment by siggy** [comment]
@farquet Merged, thanks!

**Comment by wmorgan** [comment]
Thanks @farquet! Nice to see you again!

**Pull request merged**